yellowbrick/utils/helpers.py

```diff
@@ -28,6 +28,29 @@
 
 
 ##########################################################################
+## Character Mapping helper function
+##########################################################################
+
+def character_mapping(array, mapping):
+    """
+
+    Parameters
+    ----------
+    array: iterable
+        an iterable object containing predictions from a classifier
+
+    Returns
+    -------
+    array : list
+        a list of predicted values after applying a given mapping
+
+    """
+    array = [str(x) for x in array]
```

Comment:
Any reason for mapping `str` to every element in the array? This is a lot of overhead for larger collections and should be unnecessary as far as I can tell.